libwhatami/data/quotes

```diff
@@ -39,3 +39,11 @@ If I could be precise in my description of you, I say you're a blackhole of inte
 "I love you so much. You bring joy to my life." - A filthy liar
 Seriously, I think you might need to get this checked out - why do you keep using me?
 DISCLAIMER: I am not responsible for your dumb. 
+You are a nincompoop
+You're a little pissbaby
+You think you are so cool? I think otherwise.
+Did it hurt when you fell from heaven? Because your face is pretty messed up.
+I bet you can't even read
+Go back to typing school you ignoramus!
+You make me wish I was never born
+You my friend, are the biggest asshole I have ever met
```

Comment:
Won't be including any language that might be considered inappropriate for children.

Reply:
Yeah I figured based on what you already had